observation_portal/accounts/oauth2_migrations/0002_auto_20210715_0630.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 3.2.5 on 2021-07-15 06:30
```

Comment:
Change introduced in https://github.com/observatorycontrolsystem/observation-portal/commit/d38870f9dfac56416ea4dfe26617549338ceee2c